File: AD-HOC/PYTHON/TELA_1/1026.py
while True:
    try:
        line = input()
        if line is None or line == "":
            break

        values = line.split()

        a = int(values[0])
        b = int(values[1])

        c = a ^ b

        print(c)

    except EOFError:
        break

File: AD-HOC/C#/TELA_1/1026.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string line;

        while ((line = Console.ReadLine()) != null)
        {
            string[] values = line.Split();

            uint a = uint.Parse(values[0]);
            uint b = uint.Parse(values[1]);

            uint c = a ^ b;

            Console.WriteLine(c);
        }
    }
}